rskj-core/src/main/java/co/rsk/mine/gas/provider/onchain/OnChainMinGasPriceProvider.java

```diff
@@ -0,0 +1,29 @@
+package co.rsk.mine.gas.provider.onchain;
+
+import co.rsk.config.mining.OnChainMinGasPriceSystemConfig;
+import co.rsk.mine.gas.provider.MinGasPriceProvider;
+import co.rsk.mine.gas.provider.MinGasPriceProviderType;
+import co.rsk.mine.gas.provider.StableMinGasPriceProvider;
+
+public class OnChainMinGasPriceProvider extends StableMinGasPriceProvider {
```

Comment:
We should be careful with mixing problem domains. I suspect that the RBTC/USD exchange rate != min gas price!
And so all these classes should to be renamed. Furthermore, I don't see anywhere where we make this conversion from XR to min gas price.
What am I missing?